app/src/main/java/ca/klapstein/baudit/data/Patient.java

```diff
@@ -38,4 +43,9 @@ public BodyPhoto getBodyPhoto() {
     public void setBodyPhoto(BodyPhoto bodyPhoto) {
         this.bodyPhoto = bodyPhoto;
     }
+    
+    @Override
+    public int compareTo(Patient p) {
+        return  (int)(this.getUsername().getUsername().compareTo(p.getUsername().getUsername()));
```

Comment:
casting the result of `String.compareTo(String s)` to `int` is redundant. It already returns an `int`.